src/rpc/rawtransaction.cpp

```diff
@@ -88,16 +86,21 @@ void TxToJSON(const CTransaction& tx, const uint256 hashBlock, UniValue& entry)
             in.push_back(Pair("coinbase", HexStr(txin.scriptSig.begin(), txin.scriptSig.end())));
         } else if (txin.IsZerocoinSpendV3()){
             std::unique_ptr<sigma::CoinSpendV3> spend;
+            uint32_t pubcoinId;
             try {
-                std::tie(spend, std::ignore) = ParseSigmaSpend(txin);
+                std::tie(spend, pubcoinId) = ParseSigmaSpend(txin);
             } catch (CBadTxIn&) {
                 throw JSONRPCError(RPC_DATABASE_ERROR, "An error occurred during processing the Sigma spend information");
             }
+            in.push_back(Pair("anonymityGroup", int64_t(txin.prevout.n)));
+            in.push_back(Pair("groupId", int64_t(pubcoinId)));
```

Comment:
Actually `txin.prevout.n` and `pubcoinId` will be the same value. So my idea is we should keep only `pubcoinId` but rename it name to `anonymityGroup`.